client/src/bg/background.js

```diff
@@ -4,10 +4,51 @@
 //     "sample_setting": "This is how you use Store.js to remember values"
 // });
 
+function generateUUID() {
+  let id = 'xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx'.replace(/[xy]/g, function(c) {
+    let r = Math.random()*16|0, v = c === 'x' ? r : (r&0x3|0x8);
+    return v.toString(16);
+  });
+  return id;
+}
```

Comment:
We could, but I don't think it's necessary for the current state of the app. We can be agiler if we use a simple and relatively reliable method for now, and then switch to Chrome OAuth later.

There are people thinking the same thing: https://stackoverflow.com/questions/23822170/getting-unique-clientid-from-chrome-extension#answer-23854032